docs/data/xrootd/install-standalone.md

```diff
@@ -97,9 +97,11 @@ under `/etc/xrootd/config.d/` as follows:
 
         set resourcename = UFlorida-XRD
 
-### Configuring authorization
+### Configuring authentication and authorization
 
-To configure XRootD authorization please follow the documentation [here](xrootd-authorization.md).
+XRootD offers several authentication options using security plugins to validate incoming credentials, such as bearer tokens, X.509 proxies, and VOMS proxies.
+Please follow the [XRootD authorization documentation](xrootd-authorization.md) for instructions on how to configure authorization,
+including validating credentials and mapping them to users if desired.
```

Comment:
You mention the names of _authentication_ options, but then mention the doc for configuring _authorization_, without reference to whether the authentication options are covered in the authorization doc.  Well, I suppose that's what you have in mind when you say "including validating credentials".  But do you think it would be any clearer to say this explicitly, something like this?

```suggestion
XRootD offers several authentication options using security plugins to validate incoming credentials, such as bearer tokens, X.509 proxies, and VOMS proxies.
Please follow the [XRootD authorization documentation](xrootd-authorization.md) for instructions on how to configure authentication and authorization,
including validating credentials and mapping them to users if desired.
```

Reply:
Sure